### PR TITLE
Updated  geocoder dependency from github repo.

### LINKED ISSUE #78: The plugin geocoder uses a deprecated version of the Android embedding.
**Describe the bug**
The plugin `geocoder` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.

**To Reproduce**
Compiling the app
e1

**Expected behavior**
geocoder should be replaced with [geocoder2 1.0.0 ](https://pub.dev/packages/geocoder2/install) or [geocode 1.0.1 ](https://pub.dev/packages/geocode)

**Environment**
- Device/emulator being used: <Android 10/Pixel 2 Xl>
- OS (iOS/ Android); If Android then android version: <Android: 10>
- Flutter version: <2.8>

**Are you working on this issue? (Yes/No)**
Yes, Please assign me this issue.

### PR DESCRIPTION
**Pull request by Pavel401**
Fixes #78

Describe the changes you have made in this PR -
Updated geocoder dependency from github repo.It has null safety.It compiled successfully without any errors.

```
geocoder:
   git: 
    url: https://github.com/nguyenxdat/flutter_geocoder.git 
    ref: 443b875d8ec80ce525cc6e8f17dfbbbbe7fa3028

```
Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -27,8 +27,7 @@ dependencies:
   get_it: ^7.1.3
   provider: ^5.0.0
   uni_links: ^0.5.1
-  geocoder: ^0.2.1
-  share_plus: ^3.0.4
+  share: ^2.0.4
```
Comment (nb9960) [inline pubspec.yaml:30]: Why did you replaced `share_plus` with `share`, because of this build is failing?

Comment (Pavel401) [reply pubspec.yaml]: I added it by mistake.Again I replaced it with   share_plus: ^3.0.4
.

Comment (nb9960) [reply pubspec.yaml]: Please fix the conflicting files and update your commit

File: pubspec.yaml [hunk 2]
```diff
@@ -9,43 +9,47 @@ environment:
...
   shared_preferences: ^2.0.6
-  sizer: ^2.0.15
-  skeleton_text: ^1.0.0
-  sliding_up_panel: ^2.0.0+1
+  graphql_flutter: ^5.0.0
+  get_it: ^7.1.3
+  provider: ^5.0.0
   uni_links: ^0.5.1
-
+  share_plus: ^3.0.4
```
Comment (ItsAdityaKSingh) [inline pubspec.yaml:30]: Sorted dependencies have been randomized.

Comment (Pavel401) [reply pubspec.yaml]: I have Sorted dependencies in the latest commit.

File: .env.example [hunk 3]
```diff
@@ -1,3 +0,0 @@
-MAPS_API_KEY=<your_google_maps_api_key>
-HTTP_ENDPOINT=<server's_HTTP_endpoint>(for example: https://beacon.aadibajpai.com/graphql)
-WEBSOCKET_ENDPOINT=<server's_websocket_endpoint>(for example: wss://beacon.aadibajpai.com/subscriptions)
```
Comment (nb9960) [inline .env.example:3]: Revert this change. This must not be removed

File: lib/views/home.dart [hunk 4]
```diff
@@ -83,6 +83,11 @@ class _MainScreenState extends State<MainScreen> with TickerProviderStateMixin {
                             onPressed: () => showDialog(
                                 context: context,
                                 builder: (context) => AlertDialog(
+                                      contentPadding: EdgeInsets.all(25.0),
+                                      shape: RoundedRectangleBorder(
+                                        borderRadius:
+                                            BorderRadius.circular(10.0),
+                                      ),
```
Comment (nb9960) [inline lib/views/home.dart:90]: Why this?

Comment (Pavel401) [reply lib/views/home.dart]: Added by mistake from another branch.

File: .env.example (hunk 3 above)
Comment (Pavel401) [reply .env.example:3]: I have deleted this file.

Comment (nb9960) [reply .env.example:3]: This file must not be deleted, restore it. How would beginners setup the project without .env keys

Comment (Pavel401) [reply .env.example]: I am really sorry.I have added the file again.

**Review by nb9960 — APPROVED** [review]
